components/dada-parse/src/parser.rs

```diff
@@ -63,7 +63,7 @@ impl<'me> Parser<'me> {
     /// if so, return the span and the tokens after skipping the operator.
     fn test_op(&self, op: Op) -> Option<(Span, Tokens<'me>)> {
         let mut tokens = self.tokens;
-        let span0 = tokens.last_span();
+        let span0 = tokens.peek_span();
```

Comment:
This to fix the span bug I mentioned in https://github.com/dada-lang/dada/pull/129#discussion_r806819338.  
```
fn main() {
   - 1
}
```
When calling `self.test_op(Op::Minus)`, `tokens.last_span` returns the span of `{`, and `tokens.peek_span()` returns the span of `-`